### PR TITLE
discovery: optimized code format

### PR DESCRIPTION
**Pull request by Cluas**
# Description

Please include a summary of the changes.

- fixing some typo
- optimize import
- optimize non-export variable naming

# Context

discovery package

### CODE REVIEW

File: projects/discovery/pkg/uds/syncer/setup_syncer.go [hunk 1]
```diff
@@ -64,7 +65,7 @@ func RunUDS(opts bootstrap.Opts) error {
 	emit := make(chan struct{})
 	emitter := v1.NewDiscoveryEmitterWithEmit(upstreamClient, nsClient, secretClient, emit)
 
-	// jump start all the watches
+	// jump to start all the watches
```
[the remark this answers is not in the document]
Comment (Cluas) [reply projects/discovery/pkg/uds/syncer/setup_syncer.go]: done

**Comment by Cluas** [comment]
@sam-heilbron Hi, Sam. CI build has instability, sometimes there will be a data race, and I do not know how to solve it.

**Review by nfuden — APPROVED** [review]

**Pull request merged**